src/promptflow/promptflow/_sdk/_utils.py

```diff
@@ -1128,23 +1127,61 @@ def pd_read_json(file) -> "DataFrame":
         return pd.read_json(f, orient="records", lines=True)
 
 
-def get_uuid_by_mac_id():
+def get_mac_address() -> Union[str, None]:
+    """Get the MAC ID of the first network card."""
     try:
-        node = str(uuid.getnode())
-        if node == "0":
-            raise ValueError("node value is 0")
-        return uuid.uuid3(uuid.NAMESPACE_OID, node)
+        import psutil
+
+        mac_address = None
+        net_address = psutil.net_if_addrs()
+        eth = []
+        if "Ethernet" in net_address:  # windows
+            eth = net_address["Ethernet"]
+        elif "eth0" in net_address:  # linux & mac
+            eth = net_address["eth0"]
+        for net_interface in eth:
+            if net_interface.family == psutil.AF_LINK:  # mac address
```

Comment:
waht does AF_LINK means? may add some comment here?

Reply:
AF_ LINK stands for MAC address.